logging/thirdparty/src/main/resources/application.properties

```diff
@@ -0,0 +1,17 @@
+#When you set the logging level below the minimum logging level, you must adjust the minimum logging level as well.
+#Otherwise, the value of minimum logging level overrides the logging level.
+# DOC:
+# https://access.redhat.com/documentation/en-us/red_hat_build_of_quarkus/1.11/html-single/configuring_logging_with_quarkus/index#ref-example-logging-configuration_quarkus-configuring-logging
+# https://quarkus.io/guides/logging
+# https://quarkus.io/guides/all-config#quarkus-core_quarkus.log.min-level
+
+# By default min-level is set to DEBUG
+#quarkus.log.min-level=DEBUG
+quarkus.log.level=INFO
+
+%syslog.quarkus.log.syslog.enable=true
+%syslog.quarkus.log.syslog.app-name=quarkus
+%syslog.quarkus.log.syslog.format=%-5p %s%n
+%syslog.quarkus.log.syslog.syslog-type=rfc3164
+%syslog.quarkus.log.syslog.protocol=tcp
+%syslog.quarkus.log.syslog.endpoint=localhost:1514
```

Comment:
I wonder how it works. here you set `localhost:1514` and in `@Container(port = 514)` which:

- is just internal port, not exposed port with this Quarkus communicates
- 1514 vs 514? 
- in syslog-ng.conf you use 514

If test passes, are you sure this configuration property is required for test and shouldn't be?

Reply:
no, it is overridden little later `quarkus.log.syslog.endpoint` in `withProperty`, maybe you could write here a comment that is is ignored or something? well, it is not necessary

Reply:
Added a comment